worker/README.md

```
@@ -0,0 +1,300 @@
+```bash
+cd worker
+npm install
+wrangler deploy
+```
+
+## Configuration
+
+The worker uses the following environment variables:
+
+- `BLOG_AUTHOR`: Author name for blog posts (defaults to "Paul Chris Luke")
+- `BLOG_BASE_URL`: Base URL for the blog (defaults to "https://paulchrisluke.com")
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)